tests/includes/ssh.sh

```diff
@@ -0,0 +1,33 @@
+# This file contains functions to facilitate SSH connections through a jump host.
```

Comment:
Can we add a shebang please to all shell files?

Reply:
I would've thought the same but none of the files have it so I'll leave it as is.